edx_django_utils/common/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""
+The intent of this file is that all importable public code will live here.
```

Comment:
1. This docstring's not right -- lots of code in `edx_django_utils` is publicly importable, not just this file.
2. I don't think `common` is the right name for this folder... it's very generic.  How about calling it `user` instead, since it's a user-management-specific utility?

Reply:
done